gym/spaces/space.py

```diff
@@ -7,7 +7,6 @@ class Space(object):
     action.
     """
     def __init__(self, shape=None, dtype=None):
-        import numpy as np # takes about 300-400ms to import, so we load lazily
```

Comment:
I think it's actually the import on line 1 that's undesirable (and has been added accidentally), let's rather delete that one

Reply:
fixed here, actually: https://github.com/openai/gym/pull/1400

Reply:
I see